manifests/clusterapi-controller.yaml

```diff
@@ -52,7 +52,7 @@ spec:
         image: quay.io/alberto_lamela/aws-machine-controller:dev # TODO: move this to openshift org
       {{- else if .Libvirt}}
       - name: libvirt-machine-controller
-        image: quay.io/alberto_lamela/libvirt-machine-controller:0.0.1-dev # TODO: move this to openshift org
+        image: quay.io/coreos/cluster-api-provider-libvirt:cd386e4 # TODO: move this to openshift org
```

Comment:
We should use `openshift/origin-cluster-api-provider-libvirt:v4.0.0`

Reply:
Hey @vikaschoudhary16 we need to be rebase this branch against master :)
Let's please create a PR with your fixes
Then rebase onto master